lesson_4/ex1_solution.md

```diff
@@ -1,19 +1,115 @@
 ### Part 1
+Welcome to nginx!
+If you see this page, the nginx web server is successfully installed and working. Further configuration is required.
+
+For online documentation and support please refer to nginx.org.
+Commercial support is available at nginx.com.
+
+Thank you for using nginx.
 
 
 
 
 
 ### Part 2
 
-### Part 3 logs
+Pese a que los volumenes parecen bien montados y los archivos efectivamente están donde deberían, no me muestra la página correcta ni a tiros.
```

Comment:
revisa o ex1.conf. Tes que modificalo usando o nginx.conf de dentro máis a sección "server" de lesson_4/src/nginx.conf